docs/index.md

```diff
@@ -1,15 +1,15 @@
 # Overview
 
-Molfeat is a python library to simplify molecular featurization. It supports a wide variety of molecular featurizers out-of-the-box and can be easily extended to add your own.
+Molfeat is a centralized hub of molecular featurizers. It supports a wide variety of out-of-the-box molecular featurizers and can be easily extended to include custom featurizers.
```

Comment:
```suggestion
Molfeat is a hub of molecular featurizers. It supports a wide variety of out-of-the-box molecular featurizers and can be easily extended to include your own custom featurizers.
```

Reply:
Should this be `molfeat` or `Molfeat`? I'm a bit confused going through the feedback as to how we use capitalization.